Source/TheLudoGame/Modules/Menu.cs

```diff
@@ -0,0 +1,111 @@
+﻿using GameEngine;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TheLudoGame.Context;
+
+namespace TheLudoGame.Modules
+{
+    public class Menu
+    {
+        private LudoContext ludoContext;
+
+        public Menu(LudoContext ludoContext)
+        {
+            this.ludoContext = ludoContext;
+        }
+
+        public Game GameSelectionMenu()
+        {
+            string choice = String.Empty;
+
+            while (UnsavedGameExists())
+            {
+                choice = GetChoice("Unsaved game exists. Want to load? [Y/N]");
+
+                if (choice == "Y")
+                {
+                    Console.WriteLine("Successfully loaded game.");
+                    var loadedGame = LoadGame();
+                    loadedGame.Board.Build(loadedGame.Players);
+                    Console.ReadLine();
+                    return loadedGame;
+                }
+                if (choice == "N")
```

Comment:
Snyggt att välja att logga spelen om användaren inte väljer att köra! :)

Reply:
The easy way :P

Reply:
ja, säger samma sak :) coolt!